### DISCUSSION TITLE
Find in files: no path of file in resulting panel

### DISCUSSION THREAD

**Discussion by antonkai2k** [mikekazakov/nimble-commander#227] [Ideas]
There should be a column with a path of the found file. It is impossible to understand immediately where it located.

![screenshot](https://user-images.githubusercontent.com/1384943/49591710-3da2a700-f980-11e8-91f0-11d91636c057.png)

**Comment by apd** [comment]
@mikekazakov Looks like this is implemented and the idea can be closed?

**Reply by mikekazakov** [reply]
It's still valid - there's no way to show full file paths (instead of just filenames) in the list presentation mode.

**Reply by apd** [reply]
Oh, I see. It's about panel, not search modal window.
